doc/using_pytorch.rst

```diff
@@ -566,94 +566,120 @@ The function should return a byte array of data serialized to content_type.
 The default implementation expects ``prediction`` to be a torch.Tensor and can serialize the result to JSON, CSV, or NPY.
 It accepts response content types of "application/json", "text/csv", and "application/x-npy".
 
-Working with Existing Model Data and Training Jobs
-==================================================
 
-Attach to existing training jobs
---------------------------------
+Bring your own model
+====================
 
-You can attach a PyTorch Estimator to an existing training job using the
-``attach`` method.
+You can deploy a PyTorch model that you trained outside of SageMaker by using the ``PyTorchModel`` class.
+Typically, you save a PyTorch model as a file with extension ``.pt`` or ``.pth``.
+To do this, you need to:
+
+* Write an inference script.
+* Package the model artifacts into a ``tar.gz`` file.
+* Upload the ``tar.gz`` file to an S3 bucket.
+* Create the ``PyTorchModel`` object.
+
+Write an inference script
+-------------------------
+
+You must create an inference script that implements (at least) the ``model_fn`` function that calls the loaded model to get a prediction.
+
+**Note**: If you use elastic inference with PyTorch, you can use the default ``model_fn`` implementation provided in the serving container.
+
+Optionally, you can also implement ``input_fn`` and ``output_fn`` to process input and output.
+For information about how to write an inference script, see `Serve a PyTorch Model <#serve-a-pytorch-model>`_.
+Save the inference script as ``inference.py`` in the same folder where you saved your PyTorch model.
```

Comment:
it doesn't have to be named ``inference.py`` - it can be named anything, as the name is passed through `entry_point` and then communicated to the serving container (through an environment variable)